.gitignore

```diff
@@ -2,5 +2,5 @@ dist/
 node_modules/
 yarn-error.log
 *.zip
-yarn.lock
```

Comment:
My opinion is to have a yarn.lock file, because you will generate it in any case. And it is to easy to upgrade it as you wish(only needed package).  You can see a working variant with their dependencies on dependencies version.

Reply:
I have a lot of examples with CI build fails after.

Reply:
issue is that yarn.lock will probably never be modifier each time we release new API on theia.
And we wanted to ensure it downloads the newly next version each time.

Only when it will be stabilize we could introduce back yarn.lock